core-requirements.txt

```diff
@@ -1,8 +1,8 @@
-numpy>=1.16.4
-pandas>=0.25.0
+numpy>=1.19.1
```

Comment:
If this is for ww and featuretools should we delete numpy and pandas from our core-requirements.txt?

Reply:
Interesting point... We do rely on numpy a lot so I like that we place an explicit range on the version here. Also sometimes pip can do screwy things with ordering, so its safest to list every direct dependency in the reqs files.

Reply:
Also we can always push back on / override woodwork and featuretools if we feel their requirements are overzealous

Reply:
Yea I agree listing numpy explicitly is nice but if we list a minimum version below woodwork's (or any of our dependencies) pip will install the minimum version needed for woodwork anyways. So I'm worried we'll have to keep updating our numpy dependency to match woodwork's to avoid bugs like this in the future.

Reply:
@freddyaboulton yep for sure. I hope that updates to the versions of core libs like numpy will be infrequent between our projects. Let's wait and see, and if we get feedback on this or have to make many more updates we can easily delete it.

Reply:
Sounds good @dsherry !